### DISCUSSION TITLE
ESP32 - CC1101 pinout

### DISCUSSION THREAD

**Discussion by habub** [rfquack/RFQuack#48] [Q&A]
I couldn't find any reference for wiring (pinout) when using ESP32 with CC1101.
Thanks

**Comment by phretor** [comment]
They will depend on the actual ESP32 board that you’re planning to use.

**Reply by habub** [reply]
Of course, but let's say I use the [Huzzah32]([url](docs/pinouts/featheresp32/featheresp32-pinout.png)) with the [demo app](examples/build.env/library/ESP32-CC1101-serial-build.env).
It's not clear to me which signals are connected on the ESP side and the CC1101 side:
1) RADIOA_CS=5, RADIOA_IRQ=2 - are these (5, 2) IOs of the ESP32 device or pin numbers of the Huzzah32?
2) What about the rest of the SPI signals (mosi, miso, clk)? Are they default?
3) What about the CC1101 GDOx pins? They aren't used?

Thanks

**Comment by phretor** [comment]
SPI bus pins are default and follow the labeling on each board.

CS and IRQ (GDO0) you can pick what you want and they follow the same
convention that PlatformIO board definition follows.
